package.json

```diff
@@ -37,6 +37,7 @@
     "@types/util-deprecate": "^1.0.0",
     "core-js": "^3.3.5",
     "downshift": "^5.0.0",
+    "jest": "24.9.0",
```

Comment:
Can we go to 25? They are about to release 26 now..

Reply:
react-scripts force you to have that version